tests/pyreverse/test_writer.py

```diff
@@ -140,3 +175,11 @@ def _assert_files_are_equal(generated_file):
         line for line in unified_diff(expected.splitlines(), generated.splitlines())
     )
     assert expected == generated, f"{files}{diff}"
+
+
+def test_color_for_stdlib_module(default_config):
+    writer = DiagramWriter(default_config)
+    obj = Mock()
+    obj.node = Mock()
+    obj.node.qname.return_value = "collections"
+    assert writer.get_color(obj) == "grey"
```

Comment:
I hope this test works well in the CI.
On my local computer it passes in the ``tox`` environment, but fails when running the test directly from VS code. I guess it has something to do with the way the virtual environment is set up. 
If it causes issues we can remove this test, I only included it because otherwise the corresponding could would have no test coverage. But given that the code is simple, having a more stable CI (and dev environments for other contributors) might be more important.

Reply:
It seems it's passing so probably something due to your VScode environment ? I don't see how this test could be flaky.